### PR TITLE
Modified service bindings to add route bindings

### PR DESCRIPTION
**Pull request by adamspd**

### CODE REVIEW

File: fetcher/fetcher_handlers.go [hunk 1]
```diff
@@ -192,6 +192,11 @@ func (c *Fetcher) fetchServiceBindings(session *SessionExt, entry *models.CFObje
 	if err == nil {
 		loadIndex(entry.ServiceBindings, bindings, func(r resources.ServiceCredentialBinding) string { return r.GUID })
 	}
+
+	routeBindings, _, _, err := session.V3().GetRouteBindings(LargeQuery)
```
Comment (gmllt) [inline fetcher/fetcher_handlers.go:196]: I think fetch should go through its own fetch function `fetchServiceRouteBinding` in order to take maximum advantage of the parallelization offered by the worker system.

**Review by gmllt — CHANGES_REQUESTED** [review]
I think fetch should go through its own fetch function `fetchServiceRouteBinding` in order to take maximum advantage of the parallelization offered by the worker system.

**Comment by gmllt** [comment]
Sorry, I was so wrong when we discussed the usefulness of a new metric.
Now, I see the benefit to introduce a new metric only for service_route_bindings, as service instances bound to a route are not bound to any app at the same time.

You were right, it is better to introduce a new metric to avoid the emergence of service_bindings metric which would not contain an application_id

**Review by gmllt — APPROVED** [review]
LGTM, thanks @adamspd

**Comment by benjaminguttmann-avtq** [comment]
like mentioned in the other PR please make sure to run the tests before a PR gets merged

https://github.com/cloudfoundry/cf_exporter/actions/runs/9561354046/job/26355377312

**Comment by adamspd** [comment]
> like mentioned in the other PR please make sure to run the tests before a PR gets merged
> 
> https://github.com/cloudfoundry/cf_exporter/actions/runs/9561354046/job/26355377312

I should've put the PR in draft, I was about to add it just before the merge. Thanks for the reminder.

**Pull request merged**